### PR TITLE
Fix key cache parsing

### PR DESCRIPTION
**Pull request by jthunt**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#7 key_cache related metrics is not captured with nodetool info command due to regex mismatch

#### General

- [X] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] RuboCop passes 

- [x] Existing tests pass 

#### Purpose

Update parser for nodetool to pull key cache stats

#### Known Compatibility Issues

N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 ### Added
+- Added key cache parsing for newer nodetool version
+
+### Added
```
Comment (majormoses) [inline CHANGELOG.md]: No need for this, they are both just items in an list of things added in the next release.

**Review by majormoses — CHANGES_REQUESTED** [review]
Overall looks good, lets just fix up a changelog and give us an automated (ideal) or manual testing artifact showing input and output.

**Comment by jthunt** [comment]
Updated changelog.md.    Is that what  you were looking for? 

Output

From Nodetool

> ID                     : bd3e2189-fc62-401d-822d-0a0fd7cac143
> Gossip active          : true
> Thrift active          : true
> Native Transport active: true
> Load                   : 4.71 GiB
> Generation No          : 1500659351
> Uptime (seconds)       : 419710
> Heap Memory (MB)       : 263.99 / 1004.00
> Off Heap Memory (MB)   : 31.55
> Data Center            : datacenter1
> Rack                   : rack1
> Exceptions             : 0
> Key Cache              : entries 15196, size 8.59 MB, capacity 50 MB, 146753 hits, 171136 requests, 0.858 recent hit rate, 14400 save period in seconds
> Row Cache              : entries 0, size 0 bytes, capacity 0 bytes, 0 hits, 0 requests, NaN recent hit rate, 0 save period in seconds
> Counter Cache          : entries 0, size 0 bytes, capacity 25 MB, 0 hits, 0 requests, NaN recent hit rate, 7200 save period in seconds
> Token                  : (invoke with -T/--tokens to see all 256 tokens)

From metrics-cassandra-graphite.rb

> ...
> metric-tst.cassandra.key_cache.size 9007267.84 1501079080
> metric-tst.cassandra.key_cache.capacity 52428800.0 1501079080
> metric-tst.cassandra.key_cache.hits 146753 1501079080
> metric-tst.cassandra.key_cache.requests 171136 1501079080
> metric-tst.cassandra.key_cache.hit_rate 0.858 1501079080
> ...

**Review by majormoses — APPROVED** [review]

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-cassandra/versions/1.1.0

**Pull request merged**